subprojects/launcher/src/integTest/groovy/org/gradle/launcher/continuous/ArchivesContinuousIntegrationTest.groovy

```diff
@@ -77,6 +77,7 @@ class ArchivesContinuousIntegrationTest extends Java7RequiringContinuousIntegrat
 
         buildFile << """
             task unpack(type: Sync) {
+                duplicatesStrategy = DuplicatesStrategy.INCLUDE
```

Comment:
The `Sync` task shouldn't need any different duplicate strategy, right?

Reply:
It's implicit again, yes.
Reverted.